src/sql/changes/update_2020-02-10_allow_shuffle_plots.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE projects ADD COLUMN shuffle_plots boolean;
```

Comment:
This file has the wrong year - it should be `update_2022-02-10_allow_shuffle_plots.sql`